MH4U Dex/RegionEncylcopediaCollectionViewCell.h

```diff
@@ -10,4 +10,12 @@
 
 @interface RegionEncylcopediaCollectionViewCell : UICollectionViewCell
 
+@property (nonatomic, strong) NSString *regionName;
+@property (nonatomic, strong) NSString *regionKeyName;
+
+/**
+ Update the cell's labels.
+ */
+- (void)displayContents;
```

Comment:
Our preferred pattern is to make a method like `layoutWithThing:` or `configureWithThing:` and pass, but not retain, the model object.